docs/aws/guides/lookup-tables.md

```diff
@@ -0,0 +1,126 @@
+# Lookup Tables with AWS
+
+If you use psoxy to send pseudonymized data to Worklytics and later wish to re-identify the data
+that you export from Worklytics to your premises, you'll need a lookup table in your data warehouse
+to JOIN with that data.
+
+![AWS data flow](lookup-table-data-flow.png)
+
+Our `aws-host` Terraform module, as used in our [Psoxy AWS Example](https://github.com/Worklytics/psoxy-example-aws/tree/main),
+provides a variable `lookup_table_builders` to control generation of these lookup tables.
+
+Populating this variable will generate another version of your HRIS data (aside from the one exposed
+to Worklytics) which you can then import back to your data warehouse.
+
+To enable it, add the following to your `terraform.tfvars` file:
+
+```hcl
+lookup_table_builders = {
+  "hris-lookup" = {
+    input_connector_id = "hris",
+    sanitized_accessor_role_names = [
+      # ADD LIST OF NAMES OF YOUR AWS ROLES WHICH CAN READ LOOKUP TABLE
+    ],
+    rules       = {
+      pseudonym_format = "URL_SAFE_TOKEN"
+      columnsToRedact       = [
+        "employee_email",
+        "manager_id",
+        "manager_email",
+        "join_date",
+        "leave_date"
+      ]
+      columnsToPseudonymize = [
+        "employee_id", # primary key
+      ]
+      columnsToDuplicate   = {
+        "employee_id" = "employee_id_orig"
```

Comment:
```suggestion
        "employee_id" = "employee_id_raw"
```
I'd change the name to suggest is not pseudonimized somehow.

Reply:
but that's what original means ... it's the original value, before transformation